from_dict/_from_dict.py

```diff
@@ -85,41 +85,51 @@ def get_args(t):
             return None
 
 
-def type_check(v, t) -> None:
+def type_check(check_stack: list, v: Any, t: type) -> None:
```

Comment:
Amazing work! Can we add a test that ensures the reported error stack is correct? That would make this change perfect. If you don't feel like it, let me know and I'll merge anyway - it's already good enough.

Reply:
Yes, I can add some tests.